README.md

```diff
@@ -93,6 +93,10 @@ export TRAVIS_WORKER_BUILD_API_URI='https://x:API_KEY@build-staging.travis-ci.or
 `TRAVIS_WORKER_BUILD_API_URI` can be found in the env of the job board app, e.g.:
 `heroku config:get JOB_BOARD_BUILD_API_ORG_URL -a job-board-staging`.
 
+#### Images
+
+TODO
```

Comment:
To document these a little bit (with pointer to `--help`):

* `IMAGE_ALIASES`
* `IMAGE_DEFAULT `
* `IMAGE_PROJECT_ID`
* `IMAGE_SELECTOR_TYPE`
* `IMAGE_SELECTOR_URL`
* `IMAGE_[ALIAS_]{ALIAS}`

Reply:
Ah, so at least some of these are specific to the "env" selector type, which used to be how the gce and jupiterbrain backends selected images in production.  Since the introduction of the "api" selector type, the env-based one has mostly fallen out of use on hosted Travis.  I found the placement in the `README.md` and section title confusing because I thought you were intending to address something at the top level for local development images, where I tend to think of image selection as being specific to each backend.  Sorry; rambling.   I guess I find the `Development: Running Travis Worker locally` to be confusing in that it is structured to describe steps for various backend providers, yet only the docker backend is filled in, and there's a docker-specific bit _outside_ of the docker-specific section ("Pull Docker images")...